docs/contributing/Masterlist-Editing.html

```diff
@@ -10,13 +10,33 @@ <h3>General Hints</h3>
 <ul>
   <li>Look in the masterlists for examples, and see the <a href="https://loot-api.readthedocs.io/en/stable/metadata/introduction.html">Metadata Syntax documentation</a> for the details.</li>
   <li>Only make edits/additions you're confident about - otherwise create an issue in the masterlist repository's issue tracker, or discuss in the official LOOT forum thread.</li>
-  <li>If you're using Notepad++, you can set it to use spaces instead of tabs when you press the tab key on your keyboard, avoiding YAML parsing errors. To do this, go to Settings &gt; Preferences &gt; Tab Settings &gt; Replace by space. A value of 4 is most common.</li>
   <li>Make as many silent comments as you want within the masterlists - it doesn't hurt (within reason). However, make sure to put them <em>above</em> the line(s) that they refer to.</li>
   <li>Use blank lines to separate mods that are grouped together or where it helps readability.</li>
   <li>Where you put a plugin doesn't matter - you may want to see if there are any related plugins in the masterlist already and group them together, or you can just add the plugin to the end of the list.</li>
   <li>Test your changes before uploading them, to make sure you didn't make any syntax mistakes. See <a href="Quickly-Testing-Your-Masterlist-Changes.html">Quickly Testing Your Masterlist Changes</a> for instructions.</li>
 </ul>
 
+<h4>Notepad++</h4>
+<p>If you're using <a href="https://notepad-plus-plus.org/">Notepad++</a>, you can set it to use spaces instead of tabs when you press the tab key on your keyboard, avoiding YAML parsing errors. To do this, go to Settings &gt; Preferences &gt; Languages &gt; Tab Settings &gt; yaml &gt; Replace by space. A value of 4 is most common.</p>
+
+<p>Another good advise is to make use of the keyboard shortcut "Alt+0" to collapse the main nodes of the masterlists ("common", "plugins", etc), which will make navigating the masterlists easier. The keyboard shortcut "Alt+Shift+0" will then reverse this action.</p>
```

Comment:
Grammatical suggestion

```suggestion
<p>Another good piece of advice is to make use of the keyboard shortcut "Alt+0" to collapse the main nodes of the masterlists ("common", "plugins", etc), which will make navigating the masterlists easier. The keyboard shortcut "Alt+Shift+0" will then reverse this action.</p>
```